compiler/src/dmd/dtemplate.d

```diff
@@ -5723,8 +5754,7 @@ extern (C++) final class TemplateMixin : TemplateInstance
 
 /************************************
  * This struct is needed for TemplateInstance to be the key in an associative array.
- * Fixing https://issues.dlang.org/show_bug.cgi?id=15812 and
- * https://issues.dlang.org/show_bug.cgi?id=15813 would make it unnecessary.
+ * Fixing https://issues.dlang.org/show_bug.cgi?id=15813 would make it unnecessary.
```

Comment:
This change, although correct, seems unrelated. I just want to make sure this change is intended and not the left overs of a different branch.

Reply:
It's intended, I'm updating outdated or confusing comments as I come across them. (Though I won't mix unrelated code changes of course)